app/components/post_list/post/body/acknowledgements/users_list/users_list.tsx

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useCallback, useRef} from 'react';
-import {FlatList} from 'react-native-gesture-handler';
+import {BottomSheetFlatList} from '@gorhom/bottom-sheet';
```

Comment:
Using this can be a problem when using on tablets as the bottom sheet provider is not registered. Perhaps look into hownit is done for the empji picker or other places